From c17e2d6670bff92c3baf33666339f222570eb678 Mon Sep 17 00:00:00 2001
Message-Id: <c17e2d6670bff92c3baf33666339f222570eb678.1684182345.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 15 May 2023 21:24:54 +0100
Subject: [PATCH] gnu: libmixed: Update to 2.0-1.9b2668e.

* gnu/packages/audio.scm (libmixed): Update to 2.0-1.9b2668e.
[arguments]: Enable tests.
---
 gnu/packages/audio.scm | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/gnu/packages/audio.scm b/gnu/packages/audio.scm
index e1695e7257..1ab4101138 100644
--- a/gnu/packages/audio.scm
+++ b/gnu/packages/audio.scm
@@ -3965,8 +3965,8 @@ (define-public wavpack
 
 (define-public libmixed
   ;; Release is much outdated.
-  (let ((commit "91e6b9f2438bca41205fade02c9d8f4f938838b6")
-        (revision "0"))
+  (let ((commit "9b2668e0d85175b0e92864cfbf1b9e58f77c92e0")
+        (revision "1"))
     (package
       (name "libmixed")
       (version (git-version "2.0" revision commit))
@@ -3978,14 +3978,10 @@ (define-public libmixed
                (commit commit)))
          (file-name (git-file-name name version))
          (sha256
-          (base32 "01vwgv8ivpg7a4y95krkgh656mmklsn1k3fmhwp474aj82grd3m4"))))
+          (base32 "0ql2h0hh4jl96sc9i6mk1d6qq261bvsfapinvzr9gx3lpzycpfb7"))))
       (build-system cmake-build-system)
       (arguments
        (list
-        ;; FIXME: (Sharlatan-20230326T121542+0100): Tests failed 1/34, 1 failed,
-        ;; 33 passed. There is not simple way to disable just one test.
-        ;; https://github.com/Shirakumo/libmixed/issues/13
-        #:tests? #f
         #:configure-flags
         #~(list "-DBUILD_STATIC=OFF"
                 "-DCMAKE_CXX_FLAGS=-O3 -fPIC"

base-commit: 242cc93438d67f5b35602d5add02e230850b0b43
-- 
2.39.2

